src/slack.js

```diff
@@ -0,0 +1,27 @@
+var request = require('request');
+
+function generateLine ({ author, hash, message }, repoUrl) {
+  return `      • <${repoUrl}/commit/${hash}|${hash}> - _${author}_ - *${message}*`
+}
+
+function generateSlackMessage (commits, repoUrl, projectName) {
+  const commitLines = commits.map(commit => generateLine(commit, repoUrl))
+  const header = `Deployment finished for <${repoUrl}|${projectName}>:`
+  const lines = [header, ...commitLines]
+  return lines.join('\n')
+}
+
+async function postMessage(message, webhookURL) {
+  const data = {
+    username: 'CI/CD System',
+    text: message,
+    icon_emoji: 'package'
```

Comment:
can we keep the feature for change the `icon_emoji`? :shipit:

Reply:
We can add this later. No scope addition until this is merged and in production :-)

Reply:
@rossille it's already feature provide by our bash version. And yes it's already used :stuck_out_tongue_winking_eye:

Reply:
And I've already updated the icon for `Profile` 😢